audio/music_player.asm

```diff
@@ -172,16 +172,15 @@ RenderMusicPlayer:
 	ldh [hOAMUpdate], a ; we will manually do it in LCD interrupt
 
 	ld hl, wChannelSelectorSwitches
-	ld a, NUM_MUSIC_CHANS - 1
 .ch_label_loop:
 	ld [wChannelSelector], a
 	ld a, [hli]
 	push hl
 	call DrawChannelLabel
 	pop hl
 	ld a, [wChannelSelector]
-	dec a
-	cp -1
+	inc a
```

Comment:
This makes the `wChannelSelector` iterate ascending not descending, but the values read from `wChannelSelectorSwitches` are still ascending. Is that OK?

Reply:
This was a bug, both are in ascending order, but `wChannelSelectorSwitches` was cleared entry so it always read 0